### PR TITLE
Implemented GET owl file from model with tests

### PR DESCRIPTION
**Pull request by PLangowski**

### CODE REVIEW

File: ontology_api/model/model_router.py [hunk 1]
```diff
@@ -22,12 +23,27 @@ async def create_model(self, wymyslsobieparametrjakiswejsciowybedziedobrze: int,
...
-
-        return None
+        
+    @router.post("/upload")
+    async def upload_file(self, file: UploadFile):
+        with open(file.filename, 'wb') as image:
+            content = await file.read()
+            image.write(content)
+            image.close()
+        return JSONResponse(content={"filename": file.filename},status_code=200)
```
Comment (KKucht) [inline ontology_api/model/model_router.py:33]: delete this xd

Comment (KKucht) [inline ontology_api/model/model_router.py:33]: delete this

**Review by KKucht — APPROVED** [review]
it's good work

**Review by borsuczek — APPROVED** [review]
Elegancki kodzik

**Pull request merged**